lib/node_modules/@stdlib/stats/base/dists/triangular/logpdf/benchmark/benchmark.js

```diff
@@ -84,7 +83,7 @@ bench( pkg+':factory', function benchmark( b ) {
 
 	b.tic();
 	for ( i = 0; i < b.iterations; i++ ) {
-		x = ( randu()*2.0 ) - 2.0;
+		x = uniform( -2.0, 2.0 );
```

Comment:
Can you allocate an array of values outside of the loop and then do `mylogpdf( [ x%x.length ] )`, rather than generate individual values within the loop?